section1/reflection.md

```diff
@@ -2,18 +2,62 @@
 
-2. What are the data types you learned about in this section? In your own words, define each.
+At the start of the program, I was *way* too quick to ask for help before exploring solutions myself, but now I think I've struck a decent balance. My usual workflow when encountering an error is:
+
+- Check for typos
+
+- Read the terminal messages
+
+- Attempt to work around the issue myself (the more I understand, the more likely this is to pay off)
+
+- search the Turing Slack for similar roadblocks other people faced
+
+- Google it
+
+- Make my own thread in the Slack asking for help with the issue
+
+
+ 2. What are the data types you learned about in this section? In your own words, define each.
+
+- strings: letters and other characters
+
+- numbers: integers, both positive and negative, fractions are represented with decimals (e.g. ".25")
+
+- Booleans: true or false
+
 
 3. How would you log the string `"Hello World!"` to the console?
 
+- console.log("Hello World!")
+
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
 
+- "//" are used to make a code comment, being able to leave notes for people without interfering with the code itself is essential.  
+
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
 
+- A variable is a sequence of letters, numbers or symbols that represents another value. If I had to explain it to a variable, I would say:
+"What's your name? Billy? Okay Billy, your name is like a variable. When people are talking about you, they use your name. They might say 'Billy is five years old.' When people use the name Billy, they're not talking about the *name* Billy, they're talking about you, a person named Billy. Just like names are used to talk about the people they belong to, variables are used to talk about the things they describe."
+
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
 
+- Spotify:
+
+  * timesListened: number  
+
+  * albums: string
+
+  * genre: string
+
+
 7. In your own words, explain what concatenation is.
 
+- Concatenation is the joining of two different kinds of variables (a string, numbers or Booleans) together into one variable (a string).
+
 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
 
+- YouTube might use it to store how many times you have watched videos from a particular channel, that way they can capture a string (the channel name) and a number (how many times) in the same variable.
+
 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+
+- Which is more commonly used/practical, concatenation or interpolation? 
```

Comment:
I see interpolation used more often, but I think it is really just a preference.